src/workingtitle-vcockpits-instruments-navsystems/html_ui/Pages/VCockpit/Instruments/NavSystems/AS1000/MFD/AS1000_MFD.js

```diff
@@ -13,10 +13,11 @@ class AS1000_MFD extends BaseAS1000 {
         });
         this.trackup = false;
         this.pagesContainer = this.getChildById("RightInfos");
-        this.addIndependentElementContainer(new Engine("Engine", "LeftInfos"));
+        this.engineDisplay = new Engine("Engine", "LeftInfos");
+        this.addIndependentElementContainer(this.engineDisplay);
         this.pageGroups = [
             new NavSystemPageGroup("MAP", this, [
-                new AS1000_MFD_MainMap()
+                new AS1000_MFD_MainMap(this.engineDisplay)
```

Comment:
I get that what you're doing here is trying to separate then engine logic from the map element so increase the encapsulation.   One one hand, that's a good idea.  On the other hand, it kinda breaks the (admittedly sloppy) paradigm that they wrote this stuff with.

On the whole, since we're supposed to be fixing their broken shit, I'm OK with the change. :D